contracts/compute_engines/yield_curve/YieldCurve.cairo

```diff
@@ -18,16 +17,16 @@ from contracts.admin.library import (
 
 from contracts.entry.structs import Entry
 from contracts.oracle_controller.IOracleController import IOracleController
-from contracts.oracle_implementation.IOracleImplementation import IOracleImplementation
 from contracts.compute_engines.yield_curve.structs import YieldPoint
 
 #
 # Consts
 #
 
 const DEFAULT_AGGREGATION_MODE = 0  # median
-const ON_SOURCE_KEY = 28526
+const ON_SOURCE_KEY = 28526  # str_to_felt("on")
 const FUTURE_SPOT_SOURCE_KEY = 123865098764438378875219828
```

Comment:
add comment on what this is

Reply:
https://github.com/42labs/Pontis/pull/74/commits/efd4784eb7502b80096198d57d655d012be5a3c1